cli/src/progress.rs

```diff
@@ -61,7 +61,15 @@ impl Progress {
             _ = write!(output, "{}", crossterm::cursor::Hide);
             self.guard = Some(guard);
         }
-        self.next_print = now.min(self.next_print + Duration::from_secs(1) / UPDATE_HZ);
+        let period = Duration::from_secs(1) / UPDATE_HZ;
```

Comment:
Probably this can be const?

Reply:
It'll get optimized regardless, I wouldn't overthink it.

Reply:
I left it as is (except that I inlined it again now that it's only used in one place).